bin/common.sh

```diff
@@ -187,9 +187,42 @@ function randomPassword {
   date +%s | sha256sum | base64 | head -c 32 ; echo
 }
 
+function disable_sa_token_automount {
+  local ns sa_name should_disable
+  ns=$1
+  sa_name=$2
+  should_disable=${SEC_DISABLE_SA_TOKEN_AUTOMOUNT:-true}
+
+  if [ "$should_disable" == "true" ]; then
+     log_debug "Disabling automount of API tokens for serviceAccount [$ns/$sa_name]"
+     kubectl -n $ns patch serviceAccount $sa_name -p '{"automountServiceAccountToken":false}'
+  fi
+}
+
+function patch_pod_token_automount {
+  local ns resource_type resource_name should_disable
+  ns=$1
+  resource_type=$2
+  resource_name=$3
+  should_disable=${SEC_DISABLE_SA_TOKEN_AUTOMOUNT:-true}
+
+  if [ "$should_disable" == "true" ]; then
+     log_debug "Enabling automount of API tokens for pods deployed via [$resource_type/$resource_name]"
+
+     if [ "$resource_type" == "daemonset" ] || [ "$resource_type" == "deployment" ]; then
+        kubectl -n $ns patch $resource_type  $resource_name -p '{"spec": {"template": {"spec": {"automountServiceAccountToken":true}}}}'
+     else
+        log_error "Invalid request to function [${FUNCNAME[0]}]; unsupported resource_type [$resource_type]"
+        return 1
+     fi
```

Comment:
Add a DEBUG message indicating that pod definition will NOT be patched when SEC_DISABLE_SA_TOKEN_AUTOMOUNT is set to "false"?